azure/Guidelines.md

```diff
@@ -1033,6 +1033,8 @@ For a PUT (create or replace) with additional long-running processing:
 <a name="lro-put-operation-location-includes-api-version"></a>
 :ballot_box_with_check: **YOU SHOULD** include the `api-version` query parameter in the `Operation-Location` header with the same version passed on the initial request if it is required by the get operation on the status monitor.
 
+Note: Clients may replace the value of `api-version` in the `Operation-Location` URI with a value appropriate for their application.
```

Comment:
```suggestion
Note: Clients should replace the value of `api-version` in the `Operation-Location` URI with a value appropriate for their application when calling back to the service.
```

Reply:
Chatted with Jeffrey offline and he (begrudingly) agreed to leave the sentence as-is.